content/en/case-studies/7.new-york-public-radio.md

```diff
@@ -0,0 +1,50 @@
+
+## For now you have two projects built with Nuxt, the WNYC player, and the Gothamist.
+
+Yes, the first thing we did was build the [WNYC radio page](https://www.wnyc.org/radio/) in Nuxt, it's a subset of the WNYC site where people can go to listen to the radio online to the FM stream or the AM stream. We have a stream coming in through WebSockets, and we built a couple NPM packages to help with streaming audio services. But besides that, everything else was just built in with Nuxt so it's pretty easy to do.
+
+![The Gothamist article](img/case-studies/nypr/1.png){width=736 height=407}
+
+## For the Gothamist website, did you choose Server Side Rendering or Static Generation?
+
+We chose to do an universal app, because we wanted a single page app as we are planning to add an audio player that's persistent on the bottom, that's the main reason we didn't choose SSG.
+
+## What are your next steps, do you have any intents to build another project with Nuxt?
+
+We have every intention! All our new projects are going to use Nuxt and Vue, and the next step is rebuilding the Radiolab website, it's one of our most popular shows on the radio, and besides that we're probably going to rebuild the rest of the WNYC site to use Nuxt and Vue. The Gothamist launch, a couple of months ago has gotten rave reviews. Everyone noticed that it was fast so we have the company approval now. Now we just need the time and the resources to actually start rebuilding but it is our every intention to use it for everything.
+
+## Your projects are open source, can you tell me more about it?
+
+I pushed for having [open source for all our projects](https://github.com/nypublicradio), so our design system is open source, and the Gothamist code is open source, we're hoping that we can help some other developers along the way to make their sites better and also to attract people to work on the team because we're working with some cool technologies. We built off the WNYC Radio and the design system in open source from the beginning. So if anyone was following it they saw all the problems and issues we had along the way, and how we solved them.
+
+## Would you recommend Nuxt, and for what kind of projects?
+
+Well, obviously, I'm highly gonna recommend Nuxt because I love it. At first I used to think that maybe it was overkill for small projects, but over the past couple of years I just started using it for everything so even if I have a one page, personal site, I'll still use it because it's just easier, it's faster to get it going and it's easier to work with. It's more fun than just the standard traditional Vue application so if anyone asks, I say "use it for everything. Any project".
```

Comment:
```suggestion
Well, obviously, I'm highly gonna recommend Nuxt because I love it. At first I used to think that maybe it was overkill for small projects, but over the past couple of years I just started using it for everything so even if I have a one page, personal site, I'll still use it because it's just easier, it's faster to get it going and it's easier to work with. It's more fun than just the standard traditional Vue application so if anyone asks, I say: "Use it for everything. Any project."
```